tests/unit/s2n_mem_usage_test.c

```diff
@@ -0,0 +1,167 @@
+    EXPECT_NOT_NULL(chain_and_key = s2n_cert_chain_and_key_new());
+    EXPECT_SUCCESS(s2n_cert_chain_and_key_load_pem(chain_and_key, cert_chain, private_key));
+    EXPECT_SUCCESS(s2n_config_add_cert_chain_and_key_to_store(server_config, chain_and_key));
+
+    size_t vm_data_initial = get_vm_data_size();
+    EXPECT_NOT_EQUAL(vm_data_initial, (size_t) -1);
+
+    /* Allocate all connections */
+    for (int i = 0; i < CONNECTIONS; i++)
+    {
+        /* Create nonblocking pipes */
+        EXPECT_SUCCESS(pipe(server_to_client + 2 * i));
+        EXPECT_SUCCESS(pipe(client_to_server + 2 * i));
+        for (int j = i * 2; j < (i + 1) * 2; j++) {
+            EXPECT_NOT_EQUAL(fcntl(server_to_client[j], F_SETFL, fcntl(server_to_client[j], F_GETFL) | O_NONBLOCK), -1);
+            EXPECT_NOT_EQUAL(fcntl(client_to_server[j], F_SETFL, fcntl(client_to_server[j], F_GETFL) | O_NONBLOCK), -1);
+        }
+
+        struct s2n_connection *client_conn;
+        EXPECT_NOT_NULL(client_conn = s2n_connection_new(S2N_CLIENT));
+        EXPECT_SUCCESS(s2n_connection_set_config(client_conn, client_config));
+        EXPECT_SUCCESS(s2n_connection_set_read_fd(client_conn, server_to_client[i * 2]));
+        EXPECT_SUCCESS(s2n_connection_set_write_fd(client_conn, client_to_server[i * 2 + 1]));
+        EXPECT_SUCCESS(s2n_connection_set_blinding(client_conn, S2N_SELF_SERVICE_BLINDING));
+        clients[i] = client_conn;
+
+        struct s2n_connection *server_conn;
+        EXPECT_NOT_NULL(server_conn = s2n_connection_new(S2N_SERVER));
+        EXPECT_SUCCESS(s2n_connection_set_read_fd(server_conn, client_to_server[i * 2]));
+        EXPECT_SUCCESS(s2n_connection_set_write_fd(server_conn, server_to_client[i * 2 + 1]));
+        EXPECT_SUCCESS(s2n_connection_set_config(server_conn, server_config));
+        EXPECT_SUCCESS(s2n_connection_set_blinding(server_conn, S2N_SELF_SERVICE_BLINDING));
+        servers[i] = server_conn;
+    }
+
+    size_t vm_data_after_allocation = get_vm_data_size();
+    EXPECT_NOT_EQUAL(vm_data_after_allocation, (size_t) -1);
+
+    for (int i = 0; i < CONNECTIONS; i++) {
+        EXPECT_SUCCESS(s2n_negotiate_test_server_and_client(servers[i], clients[i]));
+    }
+
+    size_t vm_data_after_handshakes = get_vm_data_size();
+    EXPECT_NOT_EQUAL(vm_data_after_handshakes, (size_t) -1);
+
+    for (int i = 0; i < CONNECTIONS; i++) {
+        EXPECT_SUCCESS(s2n_connection_free(clients[i]));
+        EXPECT_SUCCESS(s2n_connection_free(servers[i]));
+
+        for (int j = i * 2; j < (i + 1) * 2; j++) {
+            EXPECT_SUCCESS(close(server_to_client[j]));
+            EXPECT_SUCCESS(close(client_to_server[j]));
+        }
+    }
+
+    EXPECT_SUCCESS(s2n_cert_chain_and_key_free(chain_and_key));
+    EXPECT_SUCCESS(s2n_config_free(server_config));
+    EXPECT_SUCCESS(s2n_config_free(client_config));
+
+    free(cert_chain);
+    free(private_key);
+
+    fprintf(stdout, "\n");
```

Comment:
We usually don't print to stdout during Unit Tests since it [messes with the test output formatting](https://travis-ci.org/awslabs/s2n/jobs/544405776#L1634), can you comment this out?

Reply:
Sure